app.js

```diff
@@ -89,7 +89,19 @@ var renderNewOptions = function () {
         scales: {
           xAxes: [{
             type: 'linear',
-            position: 'bottom'
+            position: 'bottom',

```

Comment:
you have a bug or two in your randomizer every once in a while 1 picture will change but the other two stay the same so i would look at that, also you are getting an uncaught type error it appears to be happening at the same time so maybe you can see if you can track that down.